src/ruby/LICENSE

```diff
@@ -0,0 +1,95 @@
+Copyright (c) 2013 SymPy Development Team
```

Comment:
Do we need this LICENSE file here? All these file are already covered by our LICENSE file in the root directory.